app/javascript/src/components/Dashboard/Contacts/constant.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable import/exports-last */
+/* eslint-disable react/jsx-filename-extension */
+
+import React from "react";
+
+import { MenuHorizontal } from "neetoicons";
+import { Avatar, Typography, Dropdown } from "neetoui";
+import { v4 as uuidv4 } from "uuid";
+
+const ROW_SAMPLE_DATA1 = {
+  name_role: (
+    <div className="flex space-x-3">
+      <Avatar
+        size="medium"
+        user={{
+          name: "Ronalds Richards",
+        }}
+      />
+      <div className="flex flex-col text-gray-500">
+        <Typography style="h5">Ronalds Richards</Typography>
+        <Typography style="body3">Owner</Typography>
+      </div>
+    </div>
+  ),
+  email: <span className="text-gray-500">albert@borer.com</span>,
+  created_at: <span className="text-gray-500">Feb, 5, 2021</span>,
+  actions: (
+    <Dropdown buttonStyle="text" icon={MenuHorizontal}>
+      <li>Edit</li>
+      <li>Delete</li>
+    </Dropdown>
+  ),
+};
+
+const ROW_SAMPLE_DATA2 = {
+  name_role: (
+    <div className="flex space-x-3">
+      <Avatar
+        size="medium"
+        user={{
+          name: "Jacob Jones",
+        }}
+      />
+      <div className="flex flex-col text-gray-500">
+        <Typography style="h5">Jacob Jones</Typography>
+        <Typography style="body3">Owner</Typography>
+      </div>
+    </div>
+  ),
+  email: <span className="text-gray-500">albert@borer.com</span>,
+  created_at: <span className="text-gray-500">Feb, 5, 2021</span>,
+  actions: (
+    <Dropdown buttonStyle="text" icon={MenuHorizontal}>
+      <li>Edit</li>
+      <li>Delete</li>
+    </Dropdown>
+  ),
```

Comment:
This is a stark violation of the DRY principle we follow at BigBinary. Please create the desired reusable `render` functions.